coalib/coala.py

```diff
@@ -37,7 +37,8 @@ def main():
 
         console_printer = ConsolePrinter(print_colored=not args.no_color)
 
-        if args.json:  # needs to be checked in order to display bears in json
+        # needs to be checked in order to display bears in json
+        if args.json and not args.format:
```

Comment:
@Nosferatul We have a fundamental problem here! After...

```python
ArgumentParser().error('do not use --format and --json at ' +
                       'the same time')
```

... `coala.main()` should not continue!

@sils @Adrianzatreanu @Mixih @aptrishu @meetmangukiya